lib/faraday_middleware/response/caching.rb

```diff
@@ -66,7 +67,8 @@ def cache_key(env)
         url.query = params.any? ? build_query(params) : nil
       end
       url.normalize!
-      url.request_uri
+      
```

Comment:
There appears to be a trailing whitespace here according to Travis:

`lib/faraday_middleware/response/caching.rb:70  Line contains trailing whitespace`